src/main/java/ru/itmo/roguelike/items/TunicOfTheCyclopsKing.java

```diff
@@ -15,7 +15,7 @@
 
     @Override
     public void use(Actor actor) {
-        new MovingUpText(actor.getPosition(), "Tunic Of The Cyclops King\n55% to  resistance", Color.RED);
+        new MovingUpText(actor.getPosition(), "Tunic Of The Cyclops King\n    55% to  resistance", Color.CYAN);
```

Comment:
Maybe move this method to parent class, which will use `getName()` and `bonusSize`?

Reply:
@karl-crl

Reply:
fixed.